_includes/google_analytics.html

```diff
@@ -1,13 +1,9 @@
 <!-- the google_analytics_id gets auto inserted from the config file -->
 {% if site.google_analytics %}
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script src="https://www.googletagmanager.com/gtag/js?id=UA-277946-54" async></script>
-<script>
```

Comment:
Can you share more, what happened here? This does look suspicious indeed (duplicate UA IDs)—are we just removing it?

Reply:
Please do share :)